includes/class.llms.template.loader.php

```diff
@@ -524,12 +525,13 @@ public function maybe_prepare_post_content_restriction() {
 	 * Maybe restrict the post content in the REST loop
 	 *
 	 * @since 3.41.1
+	 * @since [version] Don't pass by reference because it's unnecessary.
 	 *
-	 * @param WP_Post  &$post  Post Object, passed by reference.
-	 * @param WP_Query &$query Query object, passed by reference.
+	 * @param WP_Post  $post  Post Object.
+	 * @param WP_Query $query Query object.
 	 * @return void
 	 */
-	public function maybe_restrict_post_content( &$post, &$query ) {
+	public function maybe_restrict_post_content( $post, $query ) {
```

Comment:
@eri-trabiccolo I didn't do a PR because I'm tired and not thinking right but do you approve this commit?